src/main/java/org/wso2/carbon/connector/connection/EmailConnection.java

```diff
@@ -0,0 +1,144 @@
+package org.wso2.carbon.connector.connection;
```

Comment:
Shall we add missing licence headers to all the files in the connector project?

Reply:
Fixed